best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,76 @@
+The best practice recommends not use more than 2 properties, production and apiUrl properties.
+To achieve that, we need to make some changes on environment files as you can see below:
+
+1 - This goes to ```environment.<dev|stage|prod>.ts```
+```
+export const environment = {
+    production: true,
+    apiUrl: 'http://www.yoursite.com'
+};
+```
```

Comment:
It would be nice to have this code within github markdown. If we are using ts, then I would expect these to be in ```typescript

For example:
**Before:**
```
export const environment = {
    production: true,
    apiUrl: 'http://www.yoursite.com'
};
```

**After:**
```typescript
export const environment = {
    production: true,
    apiUrl: 'http://www.yoursite.com'
};
```

Reply:
How we do that? It will be very nice to see the code like that.

Reply:
Got it!!! Looks cool. Thanks